packages/test/test-service-load/testConfig.json

```diff
@@ -203,20 +198,6 @@
 			"totalBlobCount": 300,
 			"blobSize": 256
 		},
-		"binarySnapshotFormat": {
```

Comment:
TYI - I was not able to find where it's used, and I think we are past the point that we need it - binary snapshots are 100% in prod and are enabled everywhere for very long time.